### ISSUE TITLE
Blank screen after logging in

### ISSUE THREAD

**Issue by logs777** [ipoddubny/webcdr#2]
Hello!
First of all thank you for beautiful solution!
I'd like to use it, but I have such problem: my OS is Centos 6, node v4.6.2, asterisk writes CDR to mysql-database, configuring call-recording is in progress.  In config.ini I've changed only db.connection parameters. Should I install specific module for working with mysql from node?
So, my problem is in blank screen, which I see after entering admin/admincdr in web-browser.
This is what I have in bash:
_[root@aster webcdr]# node server.js
22 Nov 18:26:47 - Server is now running on port 9030
GET / 302 21.628 ms - 56
Failed login attempt
GET /login 200 31.199 ms - -
GET /css/normalize.css 200 11.322 ms - -
GET /css/main.css 200 7.895 ms - -
GET /css/signin.css 200 6.367 ms - 794
GET /bower_components/bootstrap/dist/css/bootstrap.css 200 7.024 ms - -
GET /favicon.ico 200 2.588 ms - 766
POST /login 302 99.181 ms - 46
GET / 200 16.570 ms - -
GET /css/main.css 304 26.358 ms - -
GET /css/normalize.css 304 27.104 ms - -
GET /bower_components/backgrid/lib/backgrid.css 200 26.580 ms - -
GET /bower_components/backgrid-paginator/backgrid-paginator.min.css 200 12.777 ms - 847
GET /bower_components/bootstrap/dist/css/bootstrap.css 304 26.120 ms - -
GET /bower_components/bootstrap-select/bootstrap-select.min.css 200 28.108 ms - -
GET /bower_components/bootstrap-daterangepicker/daterangepicker.css 200 25.878 ms - -
GET /bower_components/font-awesome/css/font-awesome.min.css 200 22.560 ms - -
GET /css/datepicker3.css 200 8.793 ms - -
GET /css/webcdr.css 200 10.615 ms - -
GET /js/build/bundle.js 200 5.690 ms - -_

Dear @ipoddubny please, please, please can you help me? 
Also I want to mention 2 things:
1.  I have to change 2 files  to get it working: 
   ./lib/api/cdrs.js       and 
   ./lib/api/recordings.js
   In both files I added string "use strict"; in first line of file. Without these changes I get errors during "node server.js" command. May be it is only centos6-related issue?
2. Little fix for you manual in https://github.com/ipoddubny/webcdr/blob/master/INSTALL.md : 
   before running command "npm run build" user should go in upper directory with command "cd .." , without this command user will get error about "cannot find package.json".

**Comment by ipoddubny** [comment]
Thanks for the feedback! I've committed added fixes for 1 and 2. I don't think it has anything to do with CentOS, but anyway, there's no reason not to use 'use strict' in node.js.

Regarding your problem, it looks like the browser does not even try to load user profile and start the application. You can try to open DevTools (F12 on Linux/Windows or Cmd+Option+I on Macs) and look for error messages in console and/or network tab.

**Comment by logs777** [comment]
Thank you for reply.
I made screenshot from console

![webcdr - google chrome 2016-11-26 20 26 42](https://cloud.githubusercontent.com/assets/12154049/20642060/cd7dd41e-b416-11e6-8429-fb0d0269a88c.png)

also, my config.ini file is:

_[db]
client = mysql
[db.connection]
host = localhost
user = asterisk
password = somepasshere
database = asteriskcdrdb
charset=utf8
[cdr]
table = cdr
[session]
key = 123hjhfds79374gh
[web]
[recordings]
pattern = /var/spool/asterisk/monitor/%YYYY%/%MM%/%DD%/*%uniqueid%.mp3
[auth.ad]
domain = example
[auth.ad.connection]
url = ldap://server.ip.address
baseDN = dc=example,dc=org
username = cdruser@example.org
password = cdruser_ad_password_


And content of folder with audio-records:

[root@aster webcdr]# ls -la /var/spool/asterisk/monitor/2016/11/26/
total 392
-rw-r--r-- 1 asterisk asterisk  30816 Nov 26 19:57 1480179435.552.mp3
-rw-r--r-- 1 asterisk asterisk 141804 Nov 26 19:57 1480179435.552.wav
-rw-r--r-- 1 asterisk asterisk  38880 Nov 26 20:01 1480179646.554.mp3
-rw-r--r-- 1 asterisk asterisk 182124 Nov 26 20:01 1480179646.554.wav

**Comment by ipoddubny** [comment]
It looks like a dependency-of-a-dependency problem, when a wrong version of a library was installed. However, I could not reproduce the error on node.js 6.9.1 and npm v3.
You may try to remove node_modules and upgrade global npm package (`npm upgrade -g npm`), then run `npm install` followed by `npm build` again. If it doesn't help, try once again using [yarn](https://yarnpkg.com/) instead of npm.

**Comment by logs777** [comment]
Thanks a lot! 
On the page of Yarn-installation (actually, I didn't installed Yarn, I just used preinstallation steps to install npm) I found right way to install Node on Centos, may be this information will be helpful for somebody in future:
`curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -`
`yum install -y nodejs gcc-c++ make 
`
After that I decided to follow [this recommendation](https://github.com/marionettejs/backbone.marionette/issues/2559#issuecomment-192386274): to use backbone 1.2.3 instead of 1.3.2, I needed to edit package.json for that:
`vim package.json`
and replace dependency for backbone to 1.2.3 version.
Now everything works! Thank you!
